portal/app/[locale]/staking-dashboard/_components/stakeTable/index.tsx

```diff
@@ -1,17 +1,12 @@
 'use client'
 
-import {
-  ColumnDef,
-  flexRender,
-  getCoreRowModel,
-  useReactTable,
-} from '@tanstack/react-table'
-import { Card } from 'components/card'
+import { ColumnDef } from '@tanstack/react-table'
 import { ErrorBoundary } from 'components/errorBoundary'
+import { Table } from 'components/table'
+import { Header } from 'components/table/table'
```

Comment:
This import is a bit weird to read due to the double `/table/table` 😅 

What about...


```suggestion
import { Header } from 'components/table/header'
```

?

Given we're now using a `components/table` folder, perhaps we can now split each component into its own file